### DISCUSSION TITLE
Fuzzy search subdirectories recursively

### DISCUSSION THREAD

**Discussion by milesromeno** [sayanarijit/xplr#688] [Ideas]
I launch `xplr` at `~/`. I want to use `fzy` search for a specific file in a codebase nested a few directories deep within `~/`, but it looks like the `fzy` feature only searches the current directory. I type `/.eslint` knowing that my codebase contains an `.eslintrc` file, but no results are found within `xplr` because it is apparently only searching within `~/`. It would be more useful if you were able to exhaustively search within subdirectories recursively to avoid needing to manually navigate into all the sub directories first. That is actually the primary reason I looked for a TUI file explorer to begin with.

**Comment by sayanarijit** [comment]
Try this: https://github.com/sayanarijit/fzf.xplr

with `setup({recursive = true})`
